SQL/0000-00-01-Permission.sql

```diff
@@ -113,3 +113,27 @@ LOCK TABLES `user_perm_rel` WRITE, `permissions` READ;
 INSERT INTO `user_perm_rel` (userID, permID) SELECT DISTINCT 1, permID FROM permissions;
 /*!40000 ALTER TABLE `user_perm_rel` ENABLE KEYS */;
 UNLOCK TABLES;
+
+CREATE TABLE `permission_categories` (
```

Comment:
@jstirling91 perhaps better to get rid of permissions_category altogether and remove the reference in user_perm_rel since it doesn't do anything anymore that I can think of